demisto_sdk/__main__.py

```diff
@@ -346,9 +346,9 @@ def format_yml(use_git=False, file_type=None, **kwargs):
     '-h', '--help'
 )
 @click.option(
-    "-i", "--path", help="The path of an integration file or a package directory to upload", required=True)
+    "-i", "--input", help="The path of an integration file or a package directory to upload", required=True)
 @click.option(
-    "--insecure", help="Skip certificate validation", is_flag=True)
+    "-k, --insecure", help="Skip certificate validation", is_flag=True)
```

Comment:
why `k` is insecure?

Reply:
don't know. we can decide on a new name.
@anara123 WDYT?

Reply:
I think leave the `--insecure` as it is, no need for -k